source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/digit/rough_env_cfg.py

```diff
@@ -0,0 +1,353 @@
+# Copyright (c) 2022-2025, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import math
+
+from isaaclab_assets.robots.agility import get_digit_articulation_cfg
+
+from isaaclab.managers import (
+    EventTermCfg,
+    ObservationGroupCfg,
+    ObservationTermCfg,
+    RewardTermCfg,
+    SceneEntityCfg,
+    TerminationTermCfg,
+)
+from isaaclab.utils import configclass
+from isaaclab.utils.noise import AdditiveUniformNoiseCfg as Unoise
+
+import isaaclab_tasks.manager_based.locomotion.velocity.mdp as mdp
+from isaaclab_tasks.manager_based.locomotion.velocity.velocity_env_cfg import LocomotionVelocityRoughEnvCfg
+
+LEG_JOINT_NAMES = [
+    ".*_hip_roll",
+    ".*_hip_yaw",
+    ".*_hip_pitch",
+    ".*_knee",
+    ".*_toe_a",
+    ".*_toe_b",
+]
+
+
+ARM_JOINT_NAMES = [".*_arm_.*"]
```

Comment:
Might make it sense to move it to assets config of DIGIT to reuse it at other places as needed. Most asset related information should be at the same place IMO so that when we update the asset, we can change the USD and properties together.

Reply:
At least in this example, everywhere leg and arm are combined so don't see why you need it.